packages/patternfly-4/react-core/src/components/Modal/ModalBox.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly/components/ModalBox/modal-box.css';
+
+export interface ModalBoxProps extends React.HTMLProps<HTMLDivElement> {
+  /** content rendered inside the ModalBox. */
```

Comment:
We have been updating our docs to use sentence case.  Can you please update these comments.